### ISSUE TITLE
Final boss battle

### ISSUE THREAD

**Issue by TheShich** [yquake2/yquake2#834]
1) Jorg's recoil animation is missing
2) Makron's fall down animation is missing too

**Comment by BjossiAlfreds** [comment]
Those anims have a minimum damage threshold, are you using a big gun like RL or railgun?

**Comment by TheShich** [comment]
Yes, I know that I have to use RL or railgun and I did it several times. Animations are missing.

**Comment by BjossiAlfreds** [comment]
They work fine for me. Jorg has a 30% chance to do it and Makron is 35-45% depending on amount of damage. All I can think of is you are on nightmare which has no pain moves for monsters.

**Comment by TheShich** [comment]
Yes, I'm on "hard+" skill.

**Comment by Yamagi** [comment]
The difference between hard and hard+ is mostly skipping the pain animations.
